### DISCUSSION TITLE
Please add the option to USB charge while recording.

### DISCUSSION THREAD

**Discussion by unknownsolo** [gopro/labs#735] [Feature Requests]
Is there any way to add the option to charge while recording?

I use my GoPro as a dashcam, and the battery drains over time. The battery will always stay charged if we can charge it while recording every time, even for just 5 - 10 minutes during the first boot. 

Or, even better, if the GoPro can use a temp sensor to gauge whether to charge the battery, that will be awesome! 
ex: If recording & temp < 50 = charge.

I don't see why this can't be done. It will take this GoPro as a dashcam option to the next level.

**Comment by dnewman-gpsw** [comment]
This is not something Labs can do.  I believe the reason is heat, running the camera and charging the battery would use more power than the camera can shed the heat.  For Dashcam you script should detect when the camera is in motion, so it can charge when not in motion and not recording.  Turn the car on, camera boots, camera is testing GPS, or IMU for motion, it is charging until the car moves.

**Comment by unknownsolo** [comment]
That's a bummer. I want the camera to record when on, whether the car is moving or not. As much as I love GoPro Labs and all the work being done here, I might have to look at the new Ace Pro since it has all of these features right out of the box.

**Comment by dthiss** [comment]
I understand why charging and recording is not done at the same time, but isn't the camera and all features being powered from the USB?  So why does the battery drain?

**Comment by laminarize** [comment]
The go pro being unable to charge during record on USB power mode (dashcam mode) basically makes it unusable as a dashcam... Guess I will return it for a Garmin dashcam... :(

I was really excited about using a gopro.

**Reply by unknownsolo** [reply]
Check this out if you are still after an action cam/dash cam all in one: https://www.reddit.com/r/Insta360/comments/1hufy07/insta360_ace_pro_2_makes_a_great_dash_cam/

That's my post. I started with a GoPro and after not being able to make it reliably work, I switched. The "other" cam has all the options missing from GoPro. I've been very happy with it.
